app/controllers/merchant/items_controller.rb

```diff
@@ -17,4 +17,11 @@ def activate_deactivate
     end
     redirect_to merchant_user_items_path
   end
+
+  def disable_item
+    item = Item.find(params[:item_id])
+    current_user.merchant.items.delete(params[:item_id])
```

Comment:
I was looking on StackOverflow for this because in the ReadMe for the project, it says that no item is truly "deleted" but disabled.

Here is the stackoverflow question I found that most closely related: https://stackoverflow.com/questions/25660419/remove-object-from-has-many-but-dont-delete-the-original-record-in-rails/25660566

Reply:
It might just be best for us to add a column in our database holds a boolean for whether or not it's active.

Reply:
Agreed.